com.github.iwalton3.jellyfin-media-player.json

```diff
@@ -0,0 +1,275 @@
+      ],
+      "post-install": [
+        "install -Dm644 resources/meta/com.github.iwalton3.jellyfin-media-player.desktop /app/share/applications/com.github.iwalton3.jellyfin-media-player.desktop",
+        "install -Dm644 resources/images/icon.png /app/share/icons/hicolor/256x256/apps/com.github.iwalton3.jellyfin-media-player.png",
+        "install -Dm644 resources/images/icon.svg /app/share/icons/hicolor/scalable/apps/com.github.iwalton3.jellyfin-media-player.svg",
+        "install -Dm644 resources/meta/com.github.iwalton3.jellyfin-media-player.appdata.xml /app/share/metainfo/com.github.iwalton3.jellyfin-media-player.appdata.xml"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/iwalton3/jellyfin-media-player.git",
+          "commit": "4347cbd78f48fd95610a0615dec694babcdf8ac3"
+        },
+        {
+          "type": "archive",
+          "url": "https://github.com/iwalton3/jellyfin-web-jmp/releases/download/jwc-1.7.0-3/dist.zip",
+          "sha256": "4a79a6210fe0d3300da947086a6ec07f45993fffc771789c645bfac05f2cf08a",
+          "dest": "dist"
+        }
+      ],
+      "modules": [
+        {
+          "name": "mpv",
+          "only-arches": ["i386", "x86_64"],
+          "buildsystem": "simple",
+          "build-commands": [
+            "python3 waf configure --prefix=/app --enable-libmpv-shared --disable-cplayer --disable-build-date --disable-manpage-build --enable-vaapi --enable-vdpau --enable-cuda-hwaccel --enable-pulse --enable-alsa --disable-oss-audio --disable-tv --enable-uchardet",
+            "python3 waf build",
+            "python3 waf install"
+          ],
+          "cleanup": [ "/lib/pkgconfig", "/share", "/include" ],
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://github.com/mpv-player/mpv/archive/v0.29.1.tar.gz",
+              "sha256": "f9f9d461d1990f9728660b4ccb0e8cb5dce29ccaa6af567bec481b79291ca623"
+            },
+            {
+              "type": "file",
+              "url": "https://waf.io/waf-2.0.9",
+              "sha256": "2a8e0816f023995e557f79ea8940d322bec18f286917c8f9a6fa2dc3875dfa48",
+              "dest-filename": "waf"
+            }
+          ],
+          "modules": [
+            {
+              "name": "ffnvcodec",
+              "buildsystem": "simple",
+              "build-commands": [
+                  "make install PREFIX=/app"
+              ],
+              "only-arches": ["i386", "x86_64"],
+              "cleanup": [ "/lib/pkgconfig", "/include" ],
+              "sources": [
+                {
+                  "type": "archive",
+                  "url": "https://github.com/FFmpeg/nv-codec-headers/releases/download/n8.2.15.8/nv-codec-headers-8.2.15.8.tar.gz",
+                  "sha256": "770b20b7e63adf84d42394b50425cf9ac7925913ac4f77faba552aa525dd437a"
+                }
+              ]
+            },
+            {
+              "name": "ffmpeg",
```

Comment:
Also duplicated below.

Reply:
One has Nvidia hardware acceleration and only works on x86 and the other doesn't. Is there a better way to do this?

Reply:
No, that makes sense to me.